tests/queries/tests.py

```diff
@@ -3690,6 +3690,58 @@ def test_ticket_22429(self):
         self.assertSequenceEqual(queryset, [st2])
 
 
+class Ticket28297Tests(TestCase):
+    def test_join_use_correct_table_28297(self):
+        """
+        Query was assigning wrong table.
```

Comment:
Docstrings should state the expected behavior. It doesn't need to describe an old, incorrect behavior.